### ISSUE TITLE
Error: file not found

### ISSUE THREAD

**Issue by 0xh3LL0** [enzo1982/freac#21]
i download m4a audio from youtube with this: https://github.com/joeytwiddle/downloadyoutube
but when i tried to convert to mp3, it show this error
![error](https://user-images.githubusercontent.com/23132217/47478310-33ef3500-d853-11e8-8e4c-fc1171ac4931.PNG)

**Comment by enzo1982** [comment]
Your command line is missing a `-o` before the output file. Please try this:

`freaccmd -m cbr -b 96 test.m4a -o test.mp3`

This should fix it.

I'll see if I can avoid that confusing error message though.

**Comment by enzo1982** [comment]
Fixed in commit 4f70217ed71dbc19cd55119ce8adeda604357191.

**Comment by 0xh3LL0** [comment]
oh, you're right, i really miss the `-o` parameter, thanks!

**Comment by 0xh3LL0** [comment]
um, i got new issue...
![error2](https://user-images.githubusercontent.com/23132217/47538273-aa903f00-d8f4-11e8-9ed5-feab6a3d887e.PNG)

**Comment by 0xh3LL0** [comment]
using GUI only produce unplayable 1kb mp3 file

**Comment by enzo1982** [comment]
These messages come from MP4v2, the MP4 file parser used by fre:ac. They basically mean that the parser is unable to handle the MP4 file.

I came across such files a few times in the past - usually from YouTube downloads, but it never really made sense to me until today. But now I did some further investigation and found that YouTube delivers pure audio .m4a streams in an extended MP4 format called DASH. I was able to download such DASH files using youtube-dl and could reproduce the error messages.

So, unfortunately MP4v2 does not support DASH and thus is unable to decode the file. But I think I can add support for it with some effort. I will take a closer look at this in the next few weeks.

In the meantime, I'd recommend you to use the [fre:ac 1.1 alpha release](https://github.com/enzo1982/freac/releases/tag/v1.1-alpha-20180913). It falls back to another decoder when MP4v2 fails and that one is able to parse and decode the file.

The alpha release's freaccmd will accept the same command line, but you'll have to specify the CBR option in upper case (this will be fixed in an upcoming release):

`freaccmd -m CBR -b 96 test.m4a -o test.mp3`
